### PR TITLE
Handle invalid topics from clients

### LINKED ISSUE #516: Subscribing with an array throws fatal
If I pass an `array` as the argument for `autobahn`'s subscribe message the following fatal error occurs:

`PHP Fatal error:  Uncaught Error: Call to a member function add() on null in .../cboden/ratchet/src/Ratchet/Wamp/TopicManager.php:49`
 
Not that I want to pass an array, but if someone were to send one from the front end the loop would stop.

### PR DESCRIPTION
**Pull request by cboden**
The client is supposed to send a string as a topic but we're going to re-json'ify the topic in case of bad client input.

fixes #516

### CODE REVIEW

**Comment by cboden** [comment]
I think I'm going to update this and close the connection instead of supporting invalid params...

File: src/Ratchet/Wamp/ServerProtocol.php [hunk 1]
```diff
@@ -93,6 +93,10 @@ public function onMessage(ConnectionInterface $from, $msg) {
             throw new Exception("Invalid WAMP message format");
         }
 
+        if (isset($json[1]) && !is_string($json[1])) {
```
Comment (mbonneau) [inline src/Ratchet/Wamp/ServerProtocol.php]: I wasn't able to locate the spec, but is an integer a valid topic?

Comment (cboden) [reply src/Ratchet/Wamp/ServerProtocol.php]: The spec actually says full URLs but I've just ignored that.

Comment (cboden) [reply src/Ratchet/Wamp/ServerProtocol.php]: Although, even though this is now enforcing the spec this probably constitutes as a BC break. I think I'll update this to accept an int.

**Pull request merged**